org.eclipse.january/src/org/eclipse/january/dataset/LazyDatasetBase.java

```diff
@@ -232,6 +169,50 @@ public IMetadata getMetadata() {
 		return null;
 	}
 
+	@Override
```

Comment:
@PeterC-DLS, isn't the following a simpler, but equivalent, form of this method signature?
````
	public void setMetadata(MetadataType metadata) {

````